readme for containers (#1316)
deephaven/deephaven-core: containers/python-examples-redpanda/README.md

# Overview

> A Docker image for [Deephaven](https://deephaven.io/core/docs/tutorials/quickstart) with Python development, [Deephaven Examples Repository](https://github.com/deephaven/examples) and [Redpanda](https://github.com/vectorizedio/redpanda).

Reviewer: ```suggestion
A Docker image for Deephaven with Python development, [Deephaven Examples Repository](https://github.com/deephaven/examples) and [Redpanda](https://github.com/vectorizedio/redpanda). See the [README](https://github.com/deephaven/deephaven-core/blob/main/README.md#launch-python) for launch instructions.
```

## Why this is needed

> This project will open several docker containers including:

Reviewer: ```suggestion
This project will open several Docker containers, including:

```

 - grpc-api
 - web
 - grpc-proxy
 - envoy
 - examples
 - redpanda
 - apicurio-registry-mem

## Run a Docker-build job

> For full instructions to integrate [kafka with Deephaven](https://deephaven.io/core/docs/conceptual/kafka-in-deephaven/) see our [simple guide](https://deephaven.io/core/docs/how-to-guides/kafka-simple/).

Reviewer: ```suggestion
For instructions to integrate [Kafka with Deephaven](https://deephaven.io/core/docs/conceptual/kafka-in-deephaven/), see our [simple guide](https://deephaven.io/core/docs/how-to-guides/kafka-simple/).
```

> To run these images execute the following in the directory of your choice:

Reviewer: ```suggestion
To run these images, execute the following in the directory of your choice:
```

> ```

Reviewer: ```suggestion
```bash
```
